trieste/experimental/plotting/plotting_plotly.py

```diff
@@ -88,10 +87,10 @@ def add_surface_plotly(
     :return: updated plotly figure
     """
 
-    d = pd.DataFrame(f.reshape([xx.shape[0], yy.shape[1]]), index=xx, columns=yy)
+    z = f.reshape([xx.shape[0], yy.shape[1]])
```

Comment:
Does this impact the generated plots? (i.e. do we still label them correctly?)

Reply:
The generated plots look exactly the same with the same (default) labels. The x and y axes are directly provided to `go.Surface` below, so the index/columns here are redundant.